ee/api/rbac/test/test_access_control.py

```diff
@@ -343,12 +343,14 @@ def setUp(self):
 
     def test_query_counts(self):
         self._org_membership(OrganizationMembership.Level.MEMBER)
-        my_dashboard = Dashboard.objects.create(team=self.team, created_by=self.user)
-        other_user_dashboard = Dashboard.objects.create(team=self.team, created_by=self.other_user)
+        my_dashboard = Dashboard.objects.create(team=self.team, created_by=self.user, name="my dashboard")
+        other_user_dashboard = Dashboard.objects.create(
+            team=self.team, created_by=self.other_user, name="other user dashboard"
+        )
 
         # Baseline query (triggers any first time cache things)
         self.client.get(f"/api/projects/@current/notebooks/{self.notebook.short_id}")
-        baseline = 11
+        baseline = 15
```

Comment:
We make 2 more queries per dashboard as we also create a filesystem entry for it (and check that none was there already)